autoload/clap/themes/srcery.vim

```diff
@@ -0,0 +1,137 @@
+" require some tweaks to make it work.  If you are to copy this code for
+" your own color scheme, or build upon it.  Make sure that you are aware
+" of the additional tweaks for Srcery.
+
+
+let s:save_cpo = &cpoptions
+set cpoptions&vim
+
+
+" Helper Functions {{{
+
+function! s:GetHightlightColor(group, type) abort
+	if a:type ==? 'fg'
+		let l:gui_color = synIDattr(hlID(a:group), 'fg', 'gui')
+		let l:term_color = synIDattr(hlID(a:group), 'fg', 'cterm')
+	elseif a:type ==? 'bg'
+		let l:gui_color = synIDattr(hlID(a:group), 'bg', 'gui')
+		let l:term_color = synIDattr(hlID(a:group), 'bg', 'cterm')
+	endif
+
+	return [ l:gui_color, l:term_color ]
+endfunction
+
+" }}}
+" Srcery Palette {{{
+
+let s:black          = ['#1C1B19', 0]
+let s:red            = ['#EF2F27', 1]
+let s:green          = ['#519F50', 2]
+let s:yellow         = ['#FBB829', 3]
+let s:blue           = ['#2C78BF', 4]
+let s:magenta        = ['#E02C6D', 5]
+let s:cyan           = ['#0AAEB3', 6]
+let s:white          = ['#D0BFA1', 7]
+let s:bright_black   = ['#918175', 8]
+let s:bright_red     = ['#F75341', 9]
+let s:bright_green   = ['#98BC37', 10]
+let s:bright_yellow  = ['#FED06E', 11]
+let s:bright_blue    = ['#68A8E4', 12]
+let s:bright_magenta = ['#FF5C8F', 13]
+let s:bright_cyan    = ['#53FDE9', 14]
+let s:bright_white   = ['#FCE8C3', 15]
+
+" xterm colors.
+let s:orange        = ['#FF5F00', 202]
+let s:bright_orange = ['#FF8700', 208]
+let s:hard_black    = ['#121212', 233]
+let s:xgray1        = ['#262626', 235]
+let s:xgray2        = ['#303030', 236]
+let s:xgray3        = ['#3A3A3A', 237]
+let s:xgray4        = ['#444444', 238]
+let s:xgray5        = ['#4E4E4E', 239]
+let s:xgray6        = ['#585858', 240]
+
+"}}}
+" Srcery Tweaks {{{
+
+" The following tweaks is done to make sure that the theme works
+" properly. Because of reasons, we require that some additional actions
+" is taken to make sure that it works.
+"
+" We need to tweak both the SignColumn and ClapSymbol to set the colors
+" right. When using the transparent option, we explicitly needs to
+" change the symbol color.
+
+" Save original SignColumn
+let s:original_signcolumn_bg_color = s:GetHightlightColor('SignColumn', 'bg')
+
+
+" We use an augroup to load/unload tweaks needed for the team. Clap
+" includes some triggers that we can hook onto.
+
+function! ClapSrceryLoad() abort
+	" Set column color to NONE so that clap can controll this.
+	" Note!  This changes also the normal SignColumn color of the
+	" main window.  Might result in unexpected changes.
+	highlight SignColumn ctermbg=NONE guibg=NONE
+
+	" Only execute the color change when variable is set, this is to
+	" make the rounded corners with.
+	if exists('g:srcery_transparent_background')
+		let l:clap_symbol_style = 'ctermbg=' .s:black[1]. ' guibg=' .s:black[0]
+
+		execute 'highlight ClapSymbol ' . l:clap_symbol_style
+	endif
+endfunction
+
+function! ClapSrceryUnload() abort
+	" Revert the change made to SignColumn her.
```

Comment:
should be "here" not "her", also might not be necessary to specify here at all, a comment above a code snippet already implies that here refers to that snippet